packages/shylu/shylu_node/basker/src/shylubasker_def.hpp

```diff
@@ -1100,7 +1103,7 @@ namespace BaskerNS
         numeric_col_iperm_array(i) = i;
       }
 
-      #ifdef AMD_ON_D
+      #ifdef SHYLU_BASKER_AMD_ON_D
       if (btf_top_tabs_offset > 0) {
```

Comment:
I thought this would be a runtime option chosen with a parameter. This is ok for now, but we should change it to runtime option so we have the path tested with two different tests (one with AMD and one without). With compile time options, this code is not going to be tested in our nightly tests.